src/components/Observations/ObservationViews.js

```diff
@@ -93,12 +93,12 @@ const ObservationViews = ( ): Node => {
     />
   );
 
-  const renderHeader = useMemo( ( ) => (
+  const renderHeader = ( ) => (
```

Comment:
I don't think we needed `useMemo` here since the props shouldn't change much given they're a ref and a setter from `useState`

It also prevented children from re-rendering and hence the bug.